schemas/Clinical/index.js

```diff
@@ -214,6 +214,8 @@ const convertRegistrationDataToGql = data => {
   return {
     id: data._id || null,
     creator: data.creator || null,
+    fileName: data.batchName || null,
+    createdAt: data.createdAt || null,
```

Comment:
these are the only non-prettier changes right? if so, lgtm :)

Reply:
yup!